### DISCUSSION TITLE
span removed and no longer linkable?

### DISCUSSION THREAD

**Discussion by jaceklaskowski** [squidfunk/mkdocs-material#5214] [General]
Hi,

Whenever I wanted to add a linkable (referenceable) place in a page I used to add `<span id="ignoreIfExists">` and regardless of what mkdocs-material auto-generated, I could use `page/#ignoreIfExists` to jump to the place.

It stopped working a week or so ago.

What's the right way to add a marker (an anchor?) to jump to using `[ignoreIfExists](page.md#ignoreIfExists)` markdown?

Jacek

**Comment by squidfunk** [comment]
Could you please create an issue and provide a reproduction? That's not intended and likely a regression of the latest instant loading refactoring. Likely an easy fix with a good and minimal reproduction 😊

**Reply by jaceklaskowski** [reply]
Sure. I'm on it. Couldn't think of (and didn't expect) a better response! Danke schon, mein Freund!

**Reply by jaceklaskowski** [reply]
https://github.com/squidfunk/mkdocs-material/issues/5241
